src/main/java/com/github/maven_nar/Msvc.java

```diff
@@ -164,6 +170,12 @@ public void configureLinker(final LinkerDef linker) throws MojoExecutionExceptio
         for (File sdkLib : sdkLibs)
           linker.addLibraryDirectory(sdkLib, sdkArch);
 
+      final String envLib = System.getenv("LIB");
```

Comment:
As a more concrete example, here is an excerpt from my log trying to use this
Trying to build x86+x64 package, I happened to run it in a shell that was opened with VCx64 dev setup which worked fine previously, but now fails.
The additional libs in the search path result in incorrect libs being chosen and fully pathed

> [DEBUG]   (f) architecture = x86
> 
> '/MACHINE:X86'
> '/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\lib'
> '/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\atlmfc\lib'
> '/LIBPATH:C:\Program Files (x86)\Windows Kits\8.1\Lib\winv6.3\um\x86'
> '/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\lib\amd64'
> '/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\atlmfc\lib\amd64'
> '/LIBPATH:C:\Program Files (x86)\Windows Kits\8.1\Lib\winv6.3\um\x64'
> '/NOLOGO'
> 
> '"C:\Program Files (x86)\Windows Kits\8.1\Lib\winv6.3\um\x64\Secur32.Lib"'
> '"C:\Program Files (x86)\Windows Kits\8.1\Lib\winv6.3\um\x64\Crypt32.Lib"'
> 
>  error LNK2001: unresolved external symbol __imp__CryptProtectData@28
>  error LNK2001: unresolved external symbol __imp__CryptUnprotectData@28

Adding the LIBPATH or LIB environment is particularly poor for building multi targeted builds.